### PR TITLE
W-14884893 pdk jwt library gr

### PR DESCRIPTION
**Pull request by glenn-rodgers-sf**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-jwt.adoc [hunk 1]
```diff
@@ -0,0 +1,660 @@
...
+
+NOTE: To view an example policy project that uses PDK's JWT Library, see https://github.com/mulesoft/pdk-custom-policy-examples/blob/main/jwt-validation/README.md[JWT Validation Policy Example^].
+
+JSON Web Token (JWT) is a URL-secure method of representing claims to be transferred between two parties. The JWT token contains claims encoded in a JSON object as either the payload of a JSON Web Signature (JWS) or as a JSON web encryption (JWE) structure in plain text. This enables the claims to be digitally signed and protected with a message authentication code (MAC). Because the token is signed, you can trust the information and its source.
+
+The Flex Gateway Policy Development Kit (PDK) JWT library provides a set of tools to validate the JWT signatures and to extract and validate the claims of all incoming requests.
+
+== Token extraction
+
+To extract the JWT token, the library provides a `TokenProvider` enum that has to variants:
```
Comment (nledesma) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-jwt.adoc]: "to variants" => "two variants"

File: pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-jwt.adoc [hunk 2]
```diff
@@ -0,0 +1,660 @@
...
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+NOTE: To view an example policy project that uses PDK's JWT Library, see https://github.com/mulesoft/pdk-custom-policy-examples/blob/main/jwt-validation/README.md[JWT Validation Policy Example^].
+
+JSON Web Token (JWT) is a URL-secure method of representing claims to be transferred between two parties. The JWT token contains claims encoded in a JSON object as either the payload of a JSON Web Signature (JWS) or as a JSON web encryption (JWE) structure in plain text. This enables the claims to be digitally signed and protected with a message authentication code (MAC). Because the token is signed, you can trust the information and its source.
+
+The Flex Gateway Policy Development Kit (PDK) JWT library provides a set of tools to validate the JWT signatures and to extract and validate the claims of all incoming requests.
```
Comment (nledesma) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-jwt.adoc:11]: Maybe we can omit the "Flex Gateway" and simply use "Policy Development Kit".

File: pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-jwt.adoc [hunk 3]
```diff
@@ -0,0 +1,660 @@
+= Configuring JWT Library Functions
```
Comment (nledesma) [inline pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-jwt.adoc]: Can we change this for "Using JWT Library functions"?

**Comment by nledesma** [comment]
I made a few comments about the new JWT lib page. But besides that, everything looks great. Awesome job

File: pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-jwt.adoc (hunk 2 above)
Comment (glenn-rodgers-sf) [reply pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-jwt.adoc:11]: Ehhh, I actually don't think we can. My current understanding is that Flex Gateway Policy Development Kit is the official name and the PDK is the shorthand.

I agree with you, but, I think it has to stay as is.

Comment (nledesma) [reply pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-jwt.adoc:11]: No problem, it was just a suggestion

**Review by nledesma — APPROVED** [review]

**Pull request merged**